.github/workflows/run_chat_analyze.yaml

```
@@ -0,0 +1,47 @@
+name: Analyze chat trend
+
+on:
+    workflow_dispatch:
+        inputs:
+            url:
+                description: "Input URL for the Live to Analyze"
+                required: true
+                default: "https://example.com"  
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)